test/extensions/filters/http/jwt_authn/authenticator_test.cc

```diff
@@ -188,43 +193,53 @@ TEST_F(AuthenticatorTest, TestExpiredJWT) {
   auth_->verify(headers, &mock_cb_);
 }
 
-// This test verifies when a JWT is with invalid audience, JwtAudienceNotAllowed is returned.
-TEST_F(AuthenticatorTest, TestNonMatchAudJWT) {
-  EXPECT_CALL(mock_factory_ctx_.cluster_manager_, httpAsyncClientForCluster(_)).Times(0);
+// This test verifies when a JWT is not yet valid, JwtNotYetValid status is returned.
+TEST_F(AuthenticatorTest, TestNotYetValidJWT) {
+  EXPECT_CALL(*fetcher_, fetch(_, _)).Times(0);
   EXPECT_CALL(mock_cb_, onComplete(_)).WillOnce(Invoke([](const Status& status) {
```

Comment:
EXPECT_CALL(mock_cb_, onComplete(Status::JwtNotYetValid));

Reply:
Done in Done in 6ca0471